fem/fe/fe_nurbs.hpp

```diff
@@ -86,6 +86,10 @@ class NURBS1DFiniteElement : public ScalarFiniteElement,
                            DenseMatrix &dshape) const;
    virtual void CalcHessian (const IntegrationPoint &ip,
                              DenseMatrix &hessian) const;
+   virtual void Project(Coefficient &coeff,
```

Comment:
In the derived classes, `override` should be used instead of `virtual`, but `virtual` appears to be used in many derived classes. It looks like this PR is following the general pattern in MFEM which should be corrected in a future PR. I suppose we can keep this for now.

Reply:
Replaced the virtuals with override.